tools/sloths/src/components/memory/GameField.vue

```diff
@@ -133,18 +133,18 @@ export default defineComponent({
   methods: {
     getImages() {
       this.images = [
-        `${MEMORY_GAME_SRPITE}#memory01`,
-        `${MEMORY_GAME_SRPITE}#memory02`,
-        `${MEMORY_GAME_SRPITE}#memory03`,
-        `${MEMORY_GAME_SRPITE}#memory04`,
-        `${MEMORY_GAME_SRPITE}#memory05`,
-        `${MEMORY_GAME_SRPITE}#memory06`,
-        `${MEMORY_GAME_SRPITE}#memory07`,
-        `${MEMORY_GAME_SRPITE}#memory08`,
-        `${MEMORY_GAME_SRPITE}#memory09`,
-        `${MEMORY_GAME_SRPITE}#memory10`,
-        `${MEMORY_GAME_SRPITE}#memory11`,
-        `${MEMORY_GAME_SRPITE}#memory12`,
+        `${MEMORY_GAME_SPRITE}#memory01`,
+        `${MEMORY_GAME_SPRITE}#memory02`,
+        `${MEMORY_GAME_SPRITE}#memory03`,
+        `${MEMORY_GAME_SPRITE}#memory04`,
+        `${MEMORY_GAME_SPRITE}#memory05`,
+        `${MEMORY_GAME_SPRITE}#memory06`,
+        `${MEMORY_GAME_SPRITE}#memory07`,
+        `${MEMORY_GAME_SPRITE}#memory08`,
+        `${MEMORY_GAME_SPRITE}#memory09`,
+        `${MEMORY_GAME_SPRITE}#memory10`,
+        `${MEMORY_GAME_SPRITE}#memory11`,
+        `${MEMORY_GAME_SPRITE}#memory12`,
       ];
```

Comment:
```suggestion
      this.images = Array.from({ length: 12 }, (x, i) => `${MEMORY_GAME_SPRITE}#memory${i + 1}`);
```
Может вот так сделать?

Reply:
ok, done